src/plugins/console/public/application/components/storage_quota_error.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiButton, EuiSpacer, EuiButtonEmpty } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+interface Props {
+  onClearHistory: () => void;
+  onDisableSavingToHistory: () => void;
+}
+
+export const StorageQuotaError = ({ onClearHistory, onDisableSavingToHistory }: Props) => {
+  return (
+    <>
+      <EuiSpacer size="s" />
+      <EuiFlexGroup alignItems="center">
```

Comment:
Can we user `getterSize` instead of EuiSpacer ? 
![image](https://user-images.githubusercontent.com/20072247/142837330-ee780bd1-7ab8-4da2-a657-38ee393b9f13.png)